### PR TITLE
TEST: remove outdated tests from deselected_test.yaml

### PR DESCRIPTION
**Pull request by JuliaRS**
### Description

_Add a comprehensive description of proposed changes_

_List issue number(s) if exist(s): #6 (for example)_

---

Checklist to comply with before moving PR from draft:

**PR completeness and readability**

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes or created a separate PR with update and provided its number in the description, if necessary.
- [x] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_.
- [x] I have added a respective label(s) to PR if I have a permission for that.  
- [x] I have resolved any merge conflicts that might occur with the base branch.

**Testing**

- [x] The unit tests pass successfully.
- [x] I have run it locally and tested the changes extensively.

**Performance**

- [x] I have measured performance for affected algorithms using [scikit-learn_bench](https://github.com/IntelPython/scikit-learn_bench) and provided at least summary table with measured data, if performance change is expected.
- [x] I have provided justification why performance has changed or why changes are not expected.

### CODE REVIEW

**Comment by samir-nasibli** [comment]
@JuliaRS I think this is still in progress. Moving into draft

**Comment by samir-nasibli** [comment]
Please try to keep your branch updated, rebase it with main

**Comment by samir-nasibli** [comment]
@JuliaRS I would suggest just reverse use `deselected_tests` as a test cases to check. Try this experiment.
This will reduce the job time. Successfully, passed tests could be removed from deselect list

File: deselected_tests.yaml
Comment (icfaust) [inline deselected_tests.yaml:411]: @ethanglaser I don't know what exactly reduced_tests is used for (or if anything calls run_xpu_tests with 'reduced'). Will this section impact overall runtime (i.e. should we readd these to reduce CI times)? I'm leaning towards these staying in deselected_tests, but I need another opinion.

**Review by icfaust — COMMENTED** [review]
Looking good, I think some discussion of reduced_tests needs to occur before merge.

File: deselected_tests.yaml
Comment (JuliaRS) [reply deselected_tests.yaml:411]: @icfaust yes, I don't understand the difference between reduce_tests section and the ordinary deselected_tests section.

Comment (ethanglaser) [reply deselected_tests.yaml:411]: It looks like we are running with reduced flag in internal CI. @homksei do you know what the reduced section is intended for?

Comment (homksei) [reply deselected_tests.yaml:411]: Honestly, I do not have information about the reasons why this was done.

File: deselected_tests.yaml
Comment (ethanglaser) [inline deselected_tests.yaml:408]: Lets leave this comment in as well

**Comment by ethanglaser** [comment]
This is nearly ready for merge, nice work! Looks like a recent PR was merged with an update to deselected tests so please rebase and run intelci once more. I have also started two nightly jobs just in case there are some sporadic issues that haven't shown up yet:

http://intel-ci.intel.com/ef9ac180-c207-f1e5-89e5-a4bf010d0e2d
http://intel-ci.intel.com/ef9ac1ca-46d4-f1f7-9170-a4bf010d0e2d

**Review by ethanglaser — APPROVED** [review]

**Review by icfaust — APPROVED** [review]

**Review by Alexsandruss — APPROVED** [review]

**Review by samir-nasibli — APPROVED** [review]
Good to go! Thank you @JuliaRS

**Pull request merged**
